jobs/update-historical-prices.js

```diff
@@ -66,7 +66,7 @@ module.exports = async () => {
                 timestamp: new Date().setTime(timestamp),
             });
         }
-
+get
```

Comment:
is this supposed to be here?

Reply:
Uh no? I never touched that file, weird, fixed in latest commit.